src/analysis/retail/priest/holy/modules/talents/BottomRow/MiracleWorker.tsx

```diff
@@ -0,0 +1,87 @@
+import TALENTS from 'common/TALENTS/priest';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import Events, { CastEvent } from 'parser/core/Events';
+import Abilities from 'parser/core/modules/Abilities';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+
+/**
+ * Holy Word: Serenity and Holy Word: Sanctify gain an additional charge.
+ */
+class MiracleWorker extends Analyzer {
+  static dependencies = {
+    abilities: Abilities,
+    spellUsable: SpellUsable,
+  };
+
+  protected abilities!: Abilities;
+  protected spellUsable!: SpellUsable;
+
+  extraSerenityCD = 0;
+  extraSanctifyCD = 0;
+
+  constructor(options: Options) {
+    super(options);
+
+    this.active = this.selectedCombatant.hasTalent(TALENTS.MIRACLE_WORKER_TALENT.id);
+
+    this.addEventListener(
+      Events.cast.by(SELECTED_PLAYER).spell(TALENTS.HOLY_WORD_SERENITY_TALENT),
+      this.trackExtraSerenityCD,
+    );
+    this.addEventListener(
+      Events.cast.by(SELECTED_PLAYER).spell(TALENTS.HOLY_WORD_SANCTIFY_TALENT),
+      this.trackExtraSanctifyCD,
+    );
+  }
+
+  getExtraCD(spellId: number) {
+    const expectedDuration = this.spellUsable.fullCooldownDuration(spellId);
+    const chargesOnCooldown = this.spellUsable.chargesOnCooldown(spellId);
+    const remaining = this.spellUsable.cooldownRemaining(spellId);
+    if (chargesOnCooldown === 1) {
+      return remaining;
+    } else {
+      return expectedDuration - remaining;
+    }
+  }
+
+  trackExtraSerenityCD(event: CastEvent) {
+    this.extraSerenityCD += this.getExtraCD(TALENTS.HOLY_WORD_SERENITY_TALENT.id);
+  }
+
+  trackExtraSanctifyCD(event: CastEvent) {
+    this.extraSanctifyCD += this.getExtraCD(TALENTS.HOLY_WORD_SANCTIFY_TALENT.id);
+  }
+
+  statistic() {
+    const extraSerenityCasts = Math.floor(this.extraSerenityCD / 1000 / 60);
+    const extraSanctifyCasts = Math.floor(this.extraSanctifyCD / 1000 / 60);
+    return (
+      <Statistic
+        position={STATISTIC_ORDER.OPTIONAL(14)}
+        size="flexible"
+        tooltip={
+          <>
+            This statistic shows the estimated number of extra casts granted from having an
+            additional charge of Holy Word: Serenity and Holy Word: Sanctify.
+          </>
+        }
+        category={STATISTIC_CATEGORY.TALENTS}
+      >
+        <BoringSpellValueText spellId={TALENTS.MIRACLE_WORKER_TALENT.id}>
+          <>
+            {extraSerenityCasts} extra Serenity cast{extraSerenityCasts > 1 ? 's' : ''}.
+            <br />
+            {extraSanctifyCasts} extra Sanctify cast{extraSanctifyCasts > 1 ? 's' : ''}.
+          </>
```

Comment:
Use `TalentSpellText` instead of `BoringSpellValueText (do this for all of the stats) 

Also you might want to wrap the the text in `<small>` as it looks a bit large atm.

Reply:
I tried using small here but it was much too small IMO 
<img width="317" alt="image" src="https://user-images.githubusercontent.com/6947753/205776927-2c1a38ce-2b23-4747-b26c-622a49dad0e1.png">

Reply:
Tried it out using ToolTipElement and didn't really like how it looked with the dotted underline for each line. I just kept the standard statistic tooltip but used this usage of small to keep the number big
<img width="298" alt="image" src="https://user-images.githubusercontent.com/6947753/206091703-a70add2d-54a2-4247-9468-7b495dcaf964.png">
Looks good to me, thanks for the suggestion